projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -129,4 +129,14 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
       sort: parentRequest.sort
     };
   }
-}
+
+  private buildDefaultSortArg(columns: SpecificationBackedTableColumnDef[]): { key: Specification; direction: TableSortDirection } | undefined {
```

Comment:
Could we potentially use the pre-existing type from `GraphQlEntitiesQueryRequest` as the return type for this new builder function? I think it may be more readable like this:
```typescript
private buildDefaultSortArg(columns: SpecificationBackedTableColumnDef[]): typeof GraphQlEntitiesQueryRequest['sort'] {
```

Reply:
updated 😀